lib/symboldatabase.cpp

```diff
@@ -1407,7 +1407,7 @@ bool SymbolDatabase::isFunction(const Token *tok, const Scope* outerScope, const
         }
 
         // skip over const, noexcept, throw and override specifiers
```

Comment:
That comment was left untouched, but I guess things are self-explanatory here.

Reply:
I'll fix it.